src/apps/opening-hours/OpeningHoursHelpers.ts

```diff
@@ -0,0 +1,80 @@
+import dayjs from "dayjs";
+import "dayjs/locale/da";
+import weekOfYear from "dayjs/plugin/weekOfYear";
+import { DplOpeningHoursListGET200Item as OpeningHoursItem } from "../../core/dpl-cms/model";
+
+dayjs.extend(weekOfYear);
+export const getNextWeek = (date: Date): Date => {
+  return dayjs(date).add(1, "week").toDate();
+};
+
+export const getPreviousWeek = (date: Date): Date => {
+  return dayjs(date).subtract(1, "week").toDate();
+};
+
+export const getWeekDisplay = (date: Date): string => {
+  const week = dayjs(date).week();
+  const year = dayjs(date).year();
+
+  return `Uge ${week}, ${year}`;
+};
+
+const capitalizeString = (string: string): string => {
+  return string.charAt(0).toUpperCase() + string.slice(1);
+};
+
+export const formatDateToWeekday = (date: Date): string => {
+  const formattedAsWeekday = dayjs(date).locale("da").format("dddd");
+  const capitalizedWeekday = capitalizeString(formattedAsWeekday);
+
+  return capitalizedWeekday;
+};
+
+export const formatDate = (date: Date): string => {
+  return dayjs(date).locale("da").format("YYYY-MM-DD");
+};
+
+export const getWeekStartAndEndDate = (date = new Date()) => {
+  const start = dayjs(date).locale("da").startOf("week").toDate();
+  const end = dayjs(date).locale("da").endOf("week").toDate();
+
+  return { start, end };
+};
+
+export type GroupedOpeningHours = Array<{
+  day: string;
+  date: string;
+  openingHourEntries: OpeningHoursItem[];
+}>;
```

Comment:
Consider using a proper date object instead of `day` and `date`.

The property names does not say much about what they contain. The formatters are helpful but you might as well do the formatting as late as possible.

Reply:
Refactored this to use a dateTime prop that is a Date instead.